.github/scripts/Build-Windows.ps1

```diff
@@ -19,6 +16,10 @@ if ( $DebugPreference -eq 'Continue' ) {
     $InformationPreference = 'Continue'
 }
 
+if ( ! ( [System.Environment]::Is64BitOperatingSystem ) ) {
+    throw "obs-studio requires a 64bit system to build and run."
```

Comment:
In a way both, but not directly - it will build the plugin, but the plugin's CMake script will build `libobs` automatically (at least on Windows and macOS).